src/main/java/dk/kb/image/api/v1/impl/AccessApiServiceImpl.java

```diff
@@ -395,6 +408,77 @@ public StreamingOutput iIPImageRequest(
         }
     }
      
+    /* Is request classified as a thumbnail or fullsize for IIIF requests.
```

Comment:
Using the '-" af seperator now. Can see that what is _mostly_ used in ds-present and ds-discover.
But there is also use of all lowercase such as 'stoponerrors'. I prefer the - version for readability.

Reply:
There is color highlights, just missed it.

Reply:
Fair enough, but you still haven't made it a Javadoc :-)

Reply:
I have now. Is it acceptable to use javadoc link to method describing all parameters??? Repeating it again seems overkill, also because it is only called from that API method.

Reply:
Linking to methods in Javadoc can be done to either just the method `{@link Foo#bar}` or `{@link Foo#bar(String)}`. Linking with arguments makes sense where the method is overloaded and the links needs to be to a specific version.